### DISCUSSION TITLE
[Idea] Actions on modules should be logged

### DISCUSSION THREAD

**Discussion by jf-viguier** [PrestaShop/PrestaShop#37386] [Ideas]
### The problem

It's always frustrating to find the last actions before a PrestaShop crash.
Most of the time, it's caused by a module installation or upgrade.

### The solution

Log main modules actions (installation, reset, delete, ...) in the PrestaShop log.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by jf-viguier** [comment]
PR here https://github.com/PrestaShop/PrestaShop/pull/37387

**Comment by Progi1984** [comment]
Merged in 9.0.x.
